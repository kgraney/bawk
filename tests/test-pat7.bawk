
my_RP = RP;
print(my_RP);

/ foo:int1 19 38af / {
	print(foo);
	my_RP = RP;
	print(my_RP);
}

print(my_RP);
RP = my_RP;

/ foo:int1 00 / {
	print(foo);
}