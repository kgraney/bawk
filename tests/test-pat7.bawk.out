0
9
72
72
117
